### PR TITLE
IGVF-471-rm-diff-tissue-classification

### PR DESCRIPTION
**Pull request by corismall**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/in_vitro_system.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`in_vitro_system.json`*
 
+### Schema version 5
+
+* Remove `differentiated_tissue` from enums of classification.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/in_vitro_system.md:5]: please change to 
* Remove `differentiated_tissue` from `classification`.

File: src/igvfd/tests/test_types_biosample.py [hunk 2]
```diff
@@ -146,12 +146,13 @@ def test_summary(testapp, tissue, primary_cell, whole_organism, in_vitro_cell_li
...
             'biosample_term': sample_term_brown_adipose_tissue['@id'],
-            'classification': 'differentiated tissue',
+            'classification': 'organoid',
             'time_post_factors_introduction': 1,
             'time_post_factors_introduction_units': 'month'
         }
     )
     res = testapp.get(in_vitro_cell_line['@id'])
-    assert res.json.get('summary') == 'brown adipose differentiated tissue, Mus musculus (1 month)'
+    #print('SUMMARY: ', res.json.get('summary'))
```
Comment (gabdank) [inline src/igvfd/tests/test_types_biosample.py:155]: remove print statement

File: src/igvfd/tests/fixtures/schemas/in_vitro_system.py [hunk 3]
```diff
@@ -75,3 +75,13 @@ def in_vitro_system_v3(in_vitro_cell_line):
         'accession': 'IGVFSM222IIV'
     })
     return item
+
+
+@pytest.fixture
+def in_vitro_system_v4(in_vitro_organoid):
+    item = in_vitro_organoid.copy()
+    item.update({
+        'schema_version': '5',
```
Comment (gabdank) [inline src/igvfd/tests/fixtures/schemas/in_vitro_system.py:84]: I think it should be 4 and not 5?

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/tests/fixtures/schemas/in_vitro_system.py (hunk 3 above)
Comment (corismall) [reply src/igvfd/tests/fixtures/schemas/in_vitro_system.py:84]: Does that mean  i have to change the schema_version in src/igvfd/schemas/in_vitro_system.json as well?
```
"properties": {
        "schema_version": {
            "default": "5"
```

Comment (gabdank) [reply src/igvfd/tests/fixtures/schemas/in_vitro_system.py:84]: you create in_vitro_system_v4, then for some reason change the schema version of it to 5? I am not following why?

**Review by gabdank — APPROVED** [review]

**Pull request merged**